trustpinning/certs_test.go

```diff
@@ -340,6 +341,7 @@ func TestValidateRootWithPinnerCertAndIntermediates(t *testing.T) {
 	err = signed.Sign(cs, signedRoot, []data.PublicKey{ecdsax509Key}, 1, nil)
 	require.NoError(t, err)
 
+	signedRoot.Signatures[0].IsValid = true
```

Comment:
Is it necessary to set this (in any of the cases in this file) as we don't seem to check it anywhere?